pkg/cmd/pulumi/stack_change_secrets_provider.go

```diff
@@ -30,8 +30,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type StackChangeSecretsProviderCmd struct {
```

Comment:
most of these are unexported. I would keep it unexported for now.

Reply:
Ah sure